src/Trigger/RabbitMQTriggerAttributeBindingProvider.cs

```diff
@@ -17,18 +16,15 @@ internal class RabbitMQTriggerAttributeBindingProvider : ITriggerBindingProvider
     {
         private readonly INameResolver _nameResolver;
         private readonly ILogger _logger;

```

Comment:
_logger and _options aren't currently read (and I get this warning on VS) but could be used in the future. Wasn't sure if I should keep them both for now

Reply:
if they are not used, please delete. That will clean up constructor as well.